sdk/version.go

```diff
@@ -53,8 +51,24 @@ func GetVersion(dir string) (ver string, err error) {
 	return
 }
 
+func GetVersion() (string, error) {
+	return getVersion(BuildImageDir("latest"), "COREOS_VERSION=")
+}
+
+func GetVersionFromDir(dir string) (string, error) {
+	return getVersion(dir, "COREOS_VERSION=")
+}
+
 func GetLatestVersion() (string, error) {
-	return GetVersion(BuildImageDir("latest"))
+	return getVersion(BuildImageDir("latest"), "COREOS_VERSION=")
+}
+
+func GetSDKVersion() (string, error) {
+	return getVersion(BuildImageDir("latest"), "COREOS_SDK_VERSION=")
```

Comment:
I'm not sure this would ever be useful, but it would be useful to have a shorthand for getting the SDK version from `.repo/manifests`, perhaps change this to do that instead.